tests/unit.py

```diff
@@ -0,0 +1,71 @@
+# This file contains a series of unit tests that compare expected and actual javascript code generated by pyd3xter
+# instructions:
+# install pytest: http://pytest.org/latest/
+# $> python -m pytest -v unit.py
+# TODO: refactor to streamline js comparisons; consider edge cases, syntactic curbballs
+# note: expected_js field hand-written
+
+# BUG: inproper spacing
```

Comment:
These first 8 lines are basically docs. For consistency, I would move them to the README. Some minor notes.

* TODO on line 5 - inline todos are usually for specific issues, this basically just says "make the tests better". 
* line5: 'syntactic curbballs' should be 'curve balls' (typo)
* line8: should be 'improper spacing' (typo)